pkg/wandb/spec/charts/local.go

```diff
@@ -28,9 +30,40 @@ func (c LocalRelease) Validate() error {
 }
 
 func (r LocalRelease) Chart() (*chart.Chart, error) {
+	if len(r.Path) > 4 && r.Path[len(r.Path)-4:] == ".b64" {
+		return r.decodeChart()
+	}
```

Comment:
can we not directly check if the file is a string or a binary? Do we need to do it based on path?

If you really want to go this path I would do

```go
	if strings.HasSuffix(r.Path, ".b64") {
		return r.decodeChart()
	}
```